### PR TITLE
[FEAT] support inspecting subjects in stream via StreamInfo - ADR issue 91

### PR DESCRIPTION
**Pull request by aricart**

### CODE REVIEW

**Comment by aricart** [comment]
FIX #292

**Comment by aricart** [comment]
FIX #292

File: tests/jsm_test.ts [hunk 1]
```diff
@@ -984,3 +984,27 @@ Deno.test("jsm - update consumer", async () => {
...
+  const jsm = await nc.jetstreamManager();
+  const name = nuid.next();
+  await jsm.streams.add({ name, subjects: [`${name}.>`] });
+
+  const js = nc.jetstream();
+  await js.publish(`${name}.a`);
+  await js.publish(`${name}.a.b`);
+  await js.publish(`${name}.a.b.c`);
+
+  const si = await jsm.streams.info(name, { subjects_filter: `>` });
```
Comment (kozlovic) [inline tests/jsm_test.ts:1002]: I would add a test after that where you ask for info without the subjects_filter field and assert that there is no list returned. Also, I may have use a more restrictive filter such as `${name}.a.>` and expect only 2 subjects in `si.state.subjects`?

**Review by kozlovic — APPROVED** [review]
LGTM, but would expand on the test, at very least check that without filter field, si.state.subjects is not set.

File: tests/jsm_test.ts (hunk 1 above)
Comment (aricart) [reply tests/jsm_test.ts]: done.

**Review by kozlovic — APPROVED** [review]
LGTM (I suggested more restrictive than ">" as a replacement, not addition, but that's fine).

**Pull request merged**